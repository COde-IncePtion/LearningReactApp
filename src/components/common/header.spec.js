import React from "react";
import {shallow} from "enzyme";
import Header from "./header";

it("should render header", () => {
    let component = shallow(<Header/>);

    expect(component).toMatchSnapshot();
});