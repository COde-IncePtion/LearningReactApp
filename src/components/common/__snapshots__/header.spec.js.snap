// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<div>
  <ul
    className="nav nav-pills"
  >
    <li
      role="presentation"
    >
      <Link
        activeClassName="active"
        className=""
        to="app"
      >
        Home
      </Link>
    </li>
    <li
      role="presentation"
    >
      <Link
        activeClassName="active"
        className=""
        to="about"
      >
        About
      </Link>
    </li>
    <li
      role="presentation"
    >
      <Link
        activeClassName="active"
        className=""
        to="authors"
      >
        Authors
      </Link>
    </li>
  </ul>
</div>
`;
